distribution/docs/src/main/resources/content/_devComponents/metacard-type.adoc

```diff
@@ -1,7 +1,7 @@
 :title: Developing Metacard Types
 :type: developingComponent
 :status: published
-:link: _developing_metacard_types
+:link: _developing{integrating-prefix}metacard_types
```

Comment:
❓ This one looks odd. Should this be `{developing-prefix}developing_metacard_types`?

Reply:
fixed